src/components/Popover/Popover.scss

```diff
@@ -108,3 +108,7 @@ $content-max-width: rem(400px);
 .FocusTracker {
   @include visually-hidden;
 }
+
+.hideOnPrint {
```

Comment:
This works :) I noticed the other styles in this file are a little inconsistent but since this is a style for the overlay it should be `.PopoverOverlay-hideOnPrint`.

Reply:
I was definitely debating that when i wrote this lol